app/controllers/katello/api/v2/ostree_branches_controller.rb

```diff
@@ -0,0 +1,16 @@
+module Katello
+  class Api::V2::OstreeBranchesController < Api::V2::ApiController
+    apipie_concern_subst(:a_resource => N_("an ostree branch"), :resource => "ostree_branch")
+    include Katello::Concerns::Api::V2::RepositoryContentController
+
+    private
+
+    def resource_class
+      OstreeBranch
+    end
+
+    def filter_by_content_view_filter(filter)
```

Comment:
@bbuckingham @ehelms @jlsherrill @daviddavis  What would be your suggested approach to fix this

1. Add a "can_do_filters?" to every controller including this concern OR
1. Add something in the RepositoryType plugin to say "can_do_filters?"  and make the controller handle it.

Also this seems like a change that is arguably outside the scope of this PR which already seems to be doing a lot. I say this because I would have to change controllers that include this RepositoryContentController concern for either option.

 Would you prefer a separate story or want me to change here ?